bindings_arm64.go

```diff
@@ -0,0 +1,11 @@
+//go:build arm64
+// +build arm64
+
+package hashtree
+
+import (
+	"github.com/klauspost/cpuid/v2"
+)
+
+var hasShani = cpuid.CPU.Supports(cpuid.SHA2)
+var supportedCPU = true
```

Comment:
Is this supposed to be hardcoded to true?

Reply:
this is on arm we always support it